samples/msgext-northwind-inventory-csharp/lab/Exercise 03 - Run in Copilot.md

```diff
@@ -0,0 +1,108 @@
+# Building Message Extensions for Microsoft Copilot for Microsoft 365
+
+TABLE OF CONTENTS
+
+* [Welcome](./Exercise%2000%20-%20Welcome.md) 
+* [Exercise 1](./Exercise%2001%20-%20Set%20up.md) - Set up your development Environment 
+* [Exercise 2](./Exercise%2002%20-%20Run%20sample%20app.md) - Run the sample as a Message Extension
+* Exercise 3 - Run the sample as a Copilot plugin **(THIS PAGE)**
+* [Exercise 4](./Exercise%2003%20-%20Add%20a%20new%20command.md) - Add a new command
+* [Exercise 5](./Exercise%2005%20-%20Code%20tour.md) - Code tour
+
+## Exercise 2 - Run the sample as a Copilot plugin
+
+> [!NOTE]  
+> To perform the following exercise, your account must have a valid license for Copilot for Microsoft 365.
+
+## Step 1 - Test in Microsoft Copilot for Microsoft 365 (single parameter)
+
+Begin by clicking the "Try the new Teams" switch to move to the new Teams client application.
+
+> IMPORTANT - Microsoft Copilot for Microsoft 365 only works in the "New" Teams. Please don't miss this step! 
+
+> If you restart your debugger after switching to "New" teams, you may get an error message after the debugger starts. This is a known problem; please just close the error dialog and continue testing.
```

Comment:
Block quotes formatting should be surrounded by squared brackets, be prefixed with an exclamation mark and on it's own line for correct formatting to be applied.

```suggestion
> [!IMPORTANT]
> Microsoft Copilot for Microsoft 365 only works in the "New" Teams. Please don't miss this step! 
> If you restart your debugger after switching to "New" teams, you may get an error message after the debugger starts. This is a known problem; please just close the error dialog and continue testing.
```

Reply:
fixed